File: lib-database-runtime/src/main/java/com/ustadmobile/lib/database/annotation/UmSyncFindLocalChanges.java
package com.ustadmobile.lib.database.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Used by the UmSyncOutgoing method to find local changes using the local change sequence number
 * (only).
 */
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.RUNTIME)
public @interface UmSyncFindLocalChanges {
}

File: lib-database-annotations/src/commonMain/kotlin/com/ustadmobile/lib/annotation/UmSyncFindLocalChanges.kt
package com.ustadmobile.lib.database.annotation

/**
 * Used by the UmSyncOutgoing method to find local changes using the local change sequence number
 * (only).
 */
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
@Retention(AnnotationRetention.BINARY)
annotation class UmSyncFindLocalChanges
